### DISCUSSION TITLE
Multiple Spotify Connect instances?

### DISCUSSION THREAD

**Discussion by tim-weiler** [hassio-addons/addon-spotify-connect#256] [Q&A]
I currently have the Spotify Connect add-on installed.  However, I need multiple Spotify Connect instances, as they will be feeding an audio matrix for whole home audio.  I have multiple USB sound card outputs.  Is there a way to configure multiple Spotify Connect instances?  If not, is it possible to install the add-on twice, perhaps with a separate name?

Thanks!

**Comment by frenck** [comment]
Add-ons cannot be installed twice.

../Frenck
